.github/workflows/build-and-test.yaml

```diff
@@ -1,12 +1,16 @@
-name: build-and-test-scheduled
+name: build-and-test
```

Comment:
Why did you change here?

Reply:
This is because I added push events, this workflow is no longer just for scheduled jobs.

Reply:
I understand.
I'm not sure whether to add the push event trigger yet, so let's discuss it in this thread.
https://github.com/autowarefoundation/autoware.universe/pull/297#discussion_r791468797